layers/README.md

```diff
@@ -0,0 +1,85 @@
+# layers
+
+This folder contains modules and scripts for working with ATT&CK layers. The core module allows users to load, validate, process, and save ATT&CK layers. A brief overview of the components can be found below. All scripts adhere to the MITRE ATT&CK Navigator Layer file format, [version 3.0](https://github.com/mitre-attack/attack-navigator/blob/develop/layers/LAYERFORMATv3.md).
```

Comment:
While I agree with the idea that process-ing isn't a functionality of core, the documentation here isn't for core, it's for layers, which does and will contain processing stuff. I'll change the wording to manipulate though, since that's more accurate to both instances.